setup.py

```diff
@@ -34,9 +34,12 @@
 
 # Kept manually in sync with airflow.__version__
 spec = util.spec_from_file_location("airflow.version", os.path.join('airflow', 'version.py'))  # noqa
-mod = util.module_from_spec(spec)
-spec.loader.exec_module(mod)  # type: ignore
-version = mod.version  # type: ignore
+try:
+    mod = util.module_from_spec(spec)
+    spec.loader.exec_module(mod)  # type: ignore
+    version = mod.version  # type: ignore
+except Exception:  # pylint: disable=broad-except
+    version = None
```

Comment:
Can you add a comment please in the line above try as why this `try..except` block is needed

Reply:
Sure! Done.